docs/smtchecker.rst

```diff
@@ -840,4 +859,6 @@ One of such cases is the length of a dynamic storage array overflowing during a
 push: If the ``push`` operation is applied to an array of length 2^256 - 1, its
```

Comment:
Maybe it's a good idea to point to EIP-1985 in this section? It formalizes some upper bounds.